src/model/transaction/NamespaceRegistrationTransaction.ts

```diff
@@ -217,7 +217,7 @@ export class NamespaceRegistrationTransaction extends Transaction {
     protected createBuilder(): TransactionBuilder {
         let transactionBuilder: NamespaceRegistrationTransactionBuilder;
         if (this.registrationType === NamespaceRegistrationType.RootNamespace) {
-            transactionBuilder = NamespaceRegistrationTransactionBuilder.createNamespaceRegistrationTransactionBuilderRoot(
+            transactionBuilder = NamespaceRegistrationTransactionBuilder.createNamespaceRegistrationTransactionBuilderROOT(
```

Comment:
do you intend to have ROOT and CHILD capitalized in function names?

Reply:
I could change the catbuffer generator to lower case those constants like it used to be before the schema renaming. Those methods are internal in this SDK though.